pkg/sdn/plugin/node.go

```diff
@@ -98,6 +98,10 @@ func NewNodePlugin(pluginName string, osClient *osclient.Client, kClient kclient
 		return nil, nil
 	}
 
+	// If our CNI config file exists, remove it so that kubelet doesn't think
+	// we're ready yet
+	os.Remove("/etc/cni/net.d/80-openshift-sdn.conf")
```

Comment:
@danwinship no race, since we call NewNodePlugin() and thus delete the file (via BuildKubernetesNodeConfig) long before RunKubelet() gets called (whcih eventually calls NewMainKubelet() which is where network plugins get initialized).